specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/machineLearningServices.json

```diff
@@ -1781,6 +1781,489 @@
+            "description": "Error response describing why the request failed.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "EventGridFilters"
+        ],
+        "operationId": "EventGridFilters_Delete",
+        "description": "Deletes specified event grid filter.",
+        "x-ms-examples": {
+          "Delete Compute": {
+            "$ref": "./examples/deleteEventGridFilter.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/EventGridFilterNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the event grid filter got deleted."
+          },
+          "204": {
+            "description": "The event grid filter does not exist(already deleted)."
+          },
+          "default": {
+            "description": "Error response describing why the request failed.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/eventGridFilters": {
+      "get": {
+        "tags": [
+          "EventGridFilters"
+        ],
+        "operationId": "EventGridFilters_ListByWorkspace",
+        "description": "Gets event grid filters in specified workspace.",
+        "x-ms-examples": {
+          "Get Event Grid Filters": {
+            "$ref": "./examples/getEventGridFilters.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The response includes a list of event grid filters in the specified workspace",
+            "schema": {
+              "$ref": "#/definitions/EventGridFiltersList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the request failed.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/eventGridFilters/eventTypes": {
```

Comment:
>eventGridFilters/eventTypes [](start = 143, length = 27)

This path is not ARM-compliant and is breaking the following rule: [https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc004-uri-is-not-following-arm-standard-guidelines](https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc004-uri-is-not-following-arm-standard-guidelines)